[Filebeat][New Fileset] Cisco Umbrella support (#21504)
elastic/beats: filebeat/docs/modules/cisco.asciidoc

== Cisco module

This is a module for Cisco network device's logs and Cisco Umbrella. It includes the following
filesets for receiving logs over syslog or read from a file:

- `asa` fileset: supports Cisco ASA firewall logs.
- `ftd` fileset: supports Cisco Firepower Threat Defense logs.
- `ios` fileset: supports Cisco IOS router and switch logs.
- `nexus` fileset: supports Cisco Nexus switch logs.

> - `meraki` fileset: supports Cisco Meraki logs.

Reviewer: Thanks for adding this missing entry.

:fileset_ex!:

[float]
==== `umbrella` fileset settings

The Cisco Umbrella fileset primarily focuses on reading CSV files from a S3 bucket using the filebeat S3 input.

To configure Cisco Umbrella to log to either your own S3 bucket or one that is managed by Cisco please follow the https://docs.umbrella.com/deployment-umbrella/docs/log-management[Cisco Umbrella User Guide]

This fileset supports all 4 log types which is:
- Proxy
- Cloud Firewall
- IP Logs
- DNS logs

> The Cisco Umbrella fileset is dependant on that the original file path structure is to be followed. This structure is documented https://docs.umbrella.com/deployment-umbrella/docs/log-formats-and-versioning[Umbrella Log Formats and Versioning]:

Reviewer: First phrase sounds a little odd to me. What about:
```suggestion
The Cisco Umbrella fileset depends on the original file path structure being followed. This structure is documented https://docs.umbrella.com/deployment-umbrella/docs/log-formats-and-versioning[Umbrella Log Formats and Versioning]:
```

<subfolder>/<YYYY>-<MM>-<DD>/<YYYY>-<MM>-<DD>-<hh>-<mm>-<xxxx>.csv.gz
dnslogs/<year>-<month>-<day>/<year>-<month>-<day>-<hour>-<minute>.csv.gz

> When configuring the fileset, please ensure that the Queue url is set to the root folder that includes each of these subfolders above.

Reviewer: ```suggestion
When configuring the fileset, please ensure that the Queue URL is set to the root folder that includes each of these subfolders above.
```
